README.md

```diff
@@ -1,2 +1,113 @@
-# frogtek.com-site
-Frogtek company main site
+# frogtek.com
+[![Build Status](https://travis-ci.org/Frogtek/frogtek.om-site.svg)](https://travis-ci.org/Frogtek/frogtek.com-site)
+
+This is the source code for [frogtek.com](http://frogtek.com) website.
+It is based on [Jekyll](https://jekyllrb.com/) and deployed to [Github pages](https://pages.github.com)
+
+## Development
+
+You must have `ruby` and `bundler` installed
+
+```
+gem install bundler
+```
+
+To install all dependencies run:
+
+```
+bundle install
+```
+
+To run the web locally on [http://localhost:4000](http://localhost:4000)
+
+```
+bundle exec rake serve
+```
+
+To execute all tests:
+
+```
+bundle exec rake test
+```
+
+To execute only html test:
+
+```
+bundle exec rake html_lint
+```
+
+To execute only css test:
+
+```
+bundle exec rake scss_lint
+```
+
+## Contributing
+
+General rules:
+
+- Don't publish (commit to master) directly. Create a branch, do a pull request and request for a code review
+- Run the tests (and fix it) before merge to master
+- When it is possible, use markdown (.md) instead html (.html)
```

Comment:
:gb: When possible, use Markdown instead of HTML

Reply:
👍